### ISSUE TITLE
[ADE 508 Audit]: VA Notify Portal: 508 Audit Report for 43660 VA Notify Portal - Defect 2 - Critical

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#109084]
> [!NOTE]
> **QUICK SUMMARY:** Child ticket of https://github.com/department-of-veterans-affairs/va.gov-team/issues/109082 to track the work related to Defect 2 - Critical.

### Defect 2 | Critical | Created:
_WCAG 1.3.1 Information and Relationships | Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text._

### Feedback from Project Team
A - Throughout the product, the structure of information cannot be understood by assistive technology. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1)), the links that appear as a result of expanding Accessibility Testing Service 1 (Business Contact) are visually different from the other links—this group of links cannot be distinguished from the rest of the links. (New: 03/25/2025)

_The feature that was causing this error has/will be removed and replaced with a link to a new page as part of our post-audit updates._  

B - On the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Add Template -> Email), the structure of some information cannot be understood by assistive technology. For example, the contents of cell (4,2) are read as a list but the contents of cell (4,1) are not read as a list. Additionally, the cells in the second row are not read as lists. (New: 03/25/2025)
[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/email/content__;!!K2n62VMvFw!Gn52wf00pm_uaWSi4pSvv-hyoT6od3XT5gCnSH0R2NHGb7X_irffpaDL7QnZwXjqcqoc9fhACWCMkX955vuHX9LzcHk$)
(A fix for this issue is in place)

![Image](https://github.com/user-attachments/assets/e2d2258f-c4bd-4111-bd46-a23ab01ba176)

C - Throughout the product, the structure of information cannot be understood by assistive technology—text is not read as headings. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Add Template -> SMS), “Message Length” and “Personalization ((variable))” are not read as headings. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/sms/content](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/templates/new/sms/content__;!!K2n62VMvFw!BO1EktPoG92FSa3pWvKag8SzeamDjJ_chHTx-_xAgaHXIXRRVitwX9raIs5z2ikdNhS2uv2u05PiA7QUD6yHn5fsVGQ$)
(A fix for this issue is in place)

![Image](https://github.com/user-attachments/assets/048f67e0-c535-4c73-bb90-34f19b465829)

D - Throughout the product, the structure of information cannot be understood by assistive technology—text is read as unstructured text. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> Email Template 1), under Template Details accordion, the text is read as unstructured text—the identifiers are visually different from the data. (New: 03/25/2025)
_The feature that was causing this error has/will be removed and replaced with a link to a new page as part of our post-audit updates._ 

(This is the same feature mentioned in Defect 2A) 

E - Throughout the product, the table structure cannot be understood by assistive technology. For example, on the VA Notify screen (Select a service (Accessibility Testing Service 1) -> User Hub -> Users), when reading cell (5,1) with table commands, “Usernamesortable column, sorted ascending, activate to sort column descending Rolesortable column, sorted ascending, activate to sort column descending Last Loginsortable column, sorted ascending, activate to sort column descending Action a11yuser1” is read. The table headers are not read correctly and the structure of the table is not correctly identified—this is the second row. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/user_hub/users](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/42dd098a-7ad6-4e68-999d-9b0b5ce010ba/user_hub/users__;!!K2n62VMvFw!EsWSJanX4g1-3IabyclFJgtY0VYR4BTGi4oNTA0fWr0jnerzMoF2_5PqGHKtp7OiL0kAz6ww8KhvKVumgIwHnnGHESY$)

![Image](https://github.com/user-attachments/assets/154eccc4-e029-46bf-9cbc-49e669b6ffda)

 F - On the VA Notify screen (Select a service ([va.gov](http://va.gov/)) -> User Hub -> Users), the pagination links are read as part of the table. (New: 03/25/2025)

[https://staging.notifications.va.gov/services/5e5cded3-3c76-46a3-9eef-cb63589e76be/user_hub/users](https://urldefense.com/v3/__https://staging.notifications.va.gov/services/5e5cded3-3c76-46a3-9eef-cb63589e76be/user_hub/users__;!!K2n62VMvFw!A73xng9TH23JWh05nCLXHxvsQ6q8cvxVLt6Kh7tec0Rgi78VU1Tk6wQ_HrkTzA7xyqrKzi0EYmjC0GNO9paR778WFKI$)

![Image](https://github.com/user-attachments/assets/e76651c4-ac3e-4cd8-86a9-db7679a28ca7)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
